### ISSUE TITLE
Spawns bugged!

### ISSUE THREAD

**Issue by Kayade00** [poketwo/poketwo#586]
Hi! I've noticed that Poketwo isn't spawning any Pokémon in a few servers, but it works just fine in others. In one server, I spammed for a few minutes without seeing a single Pokémon, yet in another server, they were spawning normally. What could be causing this?

**Comment by WitherredAway** [comment]
There could be a couple things that may be causing this issue:
1. Check `@Poketwo config` in that server. Are the spawn channels all valid (not deleted, you have access to, etc)? Make sure they are.
2. Make sure that the bot has these permissions in **all the spawn channels** and **the channel you're sending messages in**:
    ![Screenshot_20240703_071950_Kiwi Browser](https://github.com/poketwo/poketwo/assets/81734495/e032e566-de96-441a-a544-169abb76f6c8)
    - Try giving the bot Administrator permissions for a short period of time to test if it's a permission issue.
3. There are certain criteria to what counts as a "valid message" for spawns. If it's just you sending messages, 1 spawn will occur every 24 messages with at least 1.5s between each message. 1s if multiple people. Make sure they're not poketwo commands or messages from a bot.

**Comment by Kayade00** [comment]
Hi! I've tried everything to resolve this issue. I checked the config to ensure spawns are redirected correctly and enabled all the necessary permissions shown in the image. I even granted Administrator permissions for a few minutes to confirm all channels are accessible. For more details, I'm attaching a video reference.

https://github.com/poketwo/poketwo/assets/48177192/728b108b-f0bd-4d1a-bce3-173e860ec80f
